src/main/topLevelPages/myDashboard/newVirtualContributorWizard/useNewVirtualContributorWizard.tsx

```diff
@@ -574,6 +586,28 @@ const useNewVirtualContributorWizard = (): useNewVirtualContributorWizardProvide
     }
   };
 
+  const handleCreateExternal = async (externalVcValues: ExternalVcFormValues) => {
+    if (virtualContributorInput && myAccountId) {
+      virtualContributorInput.engine = externalVcValues.engine;
+
+      virtualContributorInput.externalConfig = {
+        apiKey: externalVcValues.apiKey,
+      };
+      if (externalVcValues.assistantId) {
+        virtualContributorInput.externalConfig.assistantId = externalVcValues.assistantId;
+      }
+      const createdVc = await executeMutation({
+        values: virtualContributorInput,
+        accountId: myAccountId,
+      });
+
+      // navigate to VC page
+      if (createdVc) {
+        navigate(`vc/${createdVc.nameID}`);
+      }
+    }
```

Comment:
@bobbykolev, how best to handle that? If you point me to an example error notification I can use that'd be great!

Reply:
I'll double-check but every mutation passes apolloErrorHandler. In case of an unexpected error, there's an error toast notification in the bottom right corner. 
Looking into the code without full context - checking for createdVc before trying to navigate seems sufficient.